pandas/core/resample.py

```diff
@@ -97,6 +97,18 @@ def __getattr__(self, attr):
 
         return object.__getattribute__(self, attr)
 
+    def __iter__(self):
+        """
+        Resampler iterator
```

Comment:
Maybe a bit more explanation here.

Reply:
Do we have any example? I found only for `GroupBy`?
https://github.com/pandas-dev/pandas/blob/v0.23.4/pandas/core/groupby/groupby.py#L881-L890